src/main/java/com/group7/ecommerce/controller/admin/AdminDashboardController.java

```diff
@@ -0,0 +1,20 @@
+package com.group7.ecommerce.controller.admin;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/admin")
+public class AdminDashboardController {
```

Comment:
```suggestion
public class DashboardController {
```
vì có pacakge admin rồi nên tên controller ri là dc